### DISCUSSION TITLE
java.lang.ClassNotFoundException: kotlin.enums.EnumEntriesKt

### DISCUSSION THREAD

**Discussion by muqiuzhige** [sol4k/sol4k#70] [Q&A] [answered]
`Connection connection = new Connection("https://cold-hanni-fast-mainnet.helius-rpc.com/", Commitment.CONFIRMED);`

error：java.lang.ClassNotFoundException: kotlin.enums.EnumEntriesKt

**Comment by muqiuzhige** [comment]
version:0.4.2

**Comment by Shpota** [comment]
Hi @muqiuzhige

It looks like a dependency clash with other dependencies in your project. Could you please share your project setup (pom.xml or build.gradle) so that I can reproduce it?

**Reply by muqiuzhige** [reply]
![image](https://github.com/sol4k/sol4k/assets/11936295/df2d152f-e075-4a9d-9641-2d15eca42322)
![image](https://github.com/sol4k/sol4k/assets/11936295/0a9a3432-31dc-4448-92d1-9a38d24a5737)

**Reply by muqiuzhige** [reply]
this is all jar dependencies

**Reply by Shpota** [reply, marked as answer]
@muqiuzhige ok, outdated Kotlin dependencies seem to be the reason. Please update your Kotlin dependencies to Kotlin 1.9 or at least 1.8. Currently you have 1.4.

**Reply by muqiuzhige** [reply]
hello，my jdk version is 1.8.0_371
